src/system_structure.jl

```diff
@@ -420,8 +425,8 @@ To create a Winch:
     winch = Winch(1, TorqueControlledMachine(set), [1, 2], 100.0)
 ```
 """
-function Winch(idx, model, tether_idxs; tether_length=0.0, tether_vel=0.0)
-    return Winch(idx, model, tether_idxs, tether_length, tether_vel)
+function Winch(idx, model, tether_idxs; tether_len=0.0, tether_vel=0.0)
```

Comment:
Why do you make these names shorter? I find it more important that names are easy to read and understand, than that they are short.